sponsorship_compassion/data/payment_terms.xml

```diff
@@ -7,7 +7,6 @@
             <field name="note">BVR immediate</field>
         </record>
         <record id="payment_term_line_direct_debit" model="account.payment.term.line">
-            <field name="name">BVR immediate</field>

```

Comment:
You should completely delete this file as it was moved into contract_compassion module.